Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+pipeline {
+	options {
+		timeout(time: 40, unit: 'MINUTES')
+		buildDiscarder(logRotator(numToKeepStr:'5'))
+		timestamps()
+	}
+	agent {
+		label "centos-7"
+	}
+	tools {
+		maven 'apache-maven-latest'
+		jdk 'openjdk-jdk11-latest'
+	}
+	stages {
+		stage('Build') {
+			steps {
+				wrap([$class: 'Xvnc', useXauthority: true]) {
+					sh """
+					mvn -f pom.xml -U clean verify --batch-mode -Pbuild-individual-bundles -Pbree-libs -Ptest-on-javase-17 -Papi-check \
+          -Dmaven.repo.local=$WORKSPACE/.m2/repository \
+          -Dtycho.surefire.argLine="--add-modules ALL-SYSTEM -Dcompliance=1.8,11,17 -Djdt.performance.asserts=disabled" 
+					"""
+				}
+			}
+			post {
+				always {
+					archiveArtifacts artifacts: '*.log,*/target/work/data/.metadata/*.log,*/tests/target/work/data/.metadata/*.log,apiAnalyzer-workspace/.metadata/*.log', allowEmptyArchive: true
+					recordIssues aggregatingResults: true, enabledForFailure: true, qualityGates: [[threshold: 1, type: 'DELTA', unstable: false]], tools: [acuCobol()]
+					publishIssues issues:[scanForIssues(tool: java()), scanForIssues(tool: mavenConsole())]
+					junit '**/target/surefire-reports/*.xml'
+				}
+			}
+		}
+		stage('Check freeze period') {
```

Comment:
I have just opened PR #4 to perform the freeze-check in a dedicated GH-workflow that makes this stage obsolete.

Do you prefer to merge this PR as it is and I update #4 afterwards to remove this stage or do you prefer to update this PR and remove the stage before it is actually added to the master?

Reply:
Right now the tests were stuck in indexer once again (I've retriggered them), I hope that's not caused by any changes in the Jenkinsfile, so I would like to see this PR merged first to get clear state.

Reply:
I do agree with Andrey here. Let's complete this pr before making any other changes